main/lint-baseline.xml

```diff
@@ -0,0 +1,1570 @@
+        priority="9"
+        summary="Invalid format string"
+        explanation="If a string contains a &apos;%&apos; character, then the string may be a formatting string which will be passed to `String.format` from Java code to replace each &apos;%&apos; occurrence with specific values.&#xA;&#xA;This lint warning checks for two related problems:&#xA;(1) Formatting strings that are invalid, meaning that `String.format` will throw exceptions at runtime when attempting to use the format string.&#xA;(2) Strings containing &apos;%&apos; that are not formatting strings getting passed to a `String.format` call. In this case the &apos;%&apos; will need to be escaped as &apos;%%&apos;.&#xA;&#xA;NOTE: Not all Strings which look like formatting strings are intended for use by `String.format`; for example, they may contain date formats intended for `android.text.format.Time#format()`. Lint cannot always figure out that a String is a date format, so you may get false warnings in those scenarios. See the suppress help topic for information on how to suppress errors in that case."
+        errorLine1="        return res.getString(R.string.auth_token_explain_long, getAuthTitle());"
+        errorLine2="               ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~">
+        <location
+            file="src/cgeo/geocaching/activity/TokenAuthorizationActivity.java"
+            line="277"
+            column="16"/>
+        <location
+            file="res/values-zh-rTW/strings.xml"
+            line="824"
+            column="5"/>
+    </issue>
+
+    <issue
+        id="StringFormatMatches"
+        severity="Error"
+        message="Suspicious argument type for formatting argument #1 in `number_input_title`: conversion is `s`, received `int` (argument #2 in method call) (Did you mean formatting character `d`, &apos;o&apos; or `x`?)"
+        category="Correctness:Messages"
+        priority="9"
+        summary="`String.format` string doesn&apos;t match the XML format string"
+        explanation="This lint check ensures the following:&#xA;(1) If there are multiple translations of the format string, then all translations use the same type for the same numbered arguments&#xA;(2) The usage of the format string in Java is consistent with the format string, meaning that the parameter types passed to String.format matches those in the format string."
+        errorLine1="                        .setTitle(String.format(context.getString(R.string.number_input_title), min, max))"
+        errorLine2="                                                                                                ~~~">
+        <location
+            file="src/cgeo/geocaching/settings/ColorpickerPreference.java"
+            line="136"
+            column="97"/>
+        <location
+            file="res/values-ca/strings.xml"
+            line="1776"
+            column="5"/>
+    </issue>
+
+    <issue
+        id="StringFormatMatches"
+        severity="Error"
```

Comment:
From what I assume it means, that we should use `%1$d` placeholders instead of `%1$s` here as the the parameters `min` and `max` are integers. Correct?

That would be easy to solve and will solve the corresponding issue for each language.

Reply:
Correct., using %1$d in cs translation should fix this

Reply:
Not only in cs but in the source, whereas the translations should be updated afterwards.
Please note, this error is present for each language.

I will tackle this and push to #10157